samples/weather-forecast/resources/index.html

```diff
@@ -17,25 +17,22 @@
         border: 2px dashed gray;
         padding: 40px;
       }
+      .box h3 {
+        margin-bottom: 20px;
+      }
     </style>
   </head>
-  <body>
-    <div style="display: flex; flex-direction: column; justify-content: space-evenly;">
-      <div class="box">
-        <h3>Widget preview:</h3>
-        <br />
-        <br />
-        <section id="preview">
-            <weather-forecast></weather-forecast>
-        </section>
-      </div>
-      <div class="box">
-      <h3>Configuration preview:</h3>
-        <section id="config"></section>
-      </div>
+  <body style="display: flex">
+    <div class="box" style="flex: 2">
+      <h3>Widget preview:</h3>
+      <br />
+      <br />
+      <section id="preview">
+        <weather-forecast></weather-forecast>
```

Comment:
I wonder if this also works...
```suggestion
        <weather-forecast />
```

Reply:
No, webcompents always need the pair of tags.